libraries/AP_HAL_ChibiOS/hwdef/ARK_CANNODE/hwdef.dat

```diff
@@ -125,3 +125,8 @@ define CAN_APP_NODE_NAME "org.ardupilot.ARK_CANNODE"
 define HAL_SUPPORT_RCOUT_SERIAL 1
 define HAL_WITH_ESC_TELEM 1
 define AP_SERIALLED_ENABLED 1
+
+# Rangefinder
```

Comment:
we can accept this, but you might like to create a separate ARK-Rangefinder fw, just so users aren't presented with a wall of parameters